x/incentives/keeper/distribution_test.go

```diff
@@ -108,14 +108,17 @@ func (suite *KeeperTestSuite) TestDistributeIncentives() {
 			if tc.expPass {
 				suite.Require().NoError(err, tc.name)
 
-				// distributes the rewards to all paricpants
+				// distributes the rewards to all participants
 				sdkParticipant := sdk.AccAddress(participant.Bytes())
 				balance := suite.app.BankKeeper.GetBalance(suite.ctx, sdkParticipant, tc.denom)
-				gasRatio := float64(gasUsed) / float64(totalGasUsed)
-				coinAllocated := float64(tc.mintAmount) * (float64(allocationRate) / 100)
-				expBalance := int64(coinAllocated * gasRatio)
-				expBalance = math.MinInt64(expBalance, int64(gasUsed))
-				suite.Require().Equal(expBalance, balance.Amount.Int64(), tc.name)
+
+				gasRatio := sdk.NewDec(int64(gasUsed)).QuoInt64(int64(totalGasUsed))
+				coinAllocated := sdk.NewDec(tc.mintAmount).MulInt64(allocationRate).QuoInt64(100)
+				expBalance := coinAllocated.Mul(gasRatio)
```

Comment:
nit: I like spelling out `expected` since this can be confused with `exponential`

Reply:
`coinAllocated.Mul(gasRatio)` is the rebate? help me understand the math here a bit more